astro/src/diagrams/quickstarts/login-after-sdk.astro

```diff
@@ -0,0 +1,21 @@
+---
+import Diagram from "../../components/mermaid/Diagram.astro";
+const { alt } = Astro.props;
+
+//language=Mermaid
+const diagram = `
+sequenceDiagram
+    participant User
+    participant App as Application
+    participant FusionAuth
+
+    User ->> App : View Homepage
+    User ->> App : Click Login Link (to FusionAuth)
+    User ->> FusionAuth : View Login Form
+    FusionAuth ->> User : Show Login Form
+    User ->> FusionAuth : Fill Out and Submit Login Form
+    FusionAuth ->> FusionAuth : Authenticates User
+    FusionAuth ->> App : User is Authenticated
```

Comment:
Why did you have to remove the "redirect" steps? Isn't the process the same even with the SDK?

Reply:
Hey @vcampitelli This was modeled after the diagram in the Integrate Angular tutorial: https://fusionauth.io/docs/v1/tech/tutorials/integrate-angular
The diagram used in the other quick starts does not apply to Vue/Angular/React apps, because they do not use the FusionAuth backend do verify if the user is authenticated.